docugen/pretty_docs.py

```diff
@@ -533,39 +549,15 @@ def _build_signature(
     return "".join(parts)
 
 
-TABLE_HEADER = (
-    # '<table class="tfo-notebook-buttons tfo-api nocontent" align="left">'
-    ""
-)
-
 _TABLE_TEMPLATE = textwrap.dedent(
     """
-    {table_header}
     {table_content}
-
     {table_footer}"""
 )
 
-# _TABLE_LINK_TEMPLATE = (
-#     "[![](https://www.tensorflow.org/images/GitHub-Mark-32px.png)"
-#     " View source on GitHub]({url})"
-# )
 
 _TABLE_LINK_TEMPLATE = (
-    "<p>"
-    "<button style={{{{display: 'flex', alignItems: 'center', "
-    "backgroundColor: 'white', border: '1px solid #ddd', padding: '10px', "
-    "borderRadius: '6px', cursor: 'pointer', "
-    "boxShadow: '0 2px 3px rgba(0,0,0,0.1)', transition: 'all 0.3s'}}}}>"
-    "<a href='{url}' style={{{{fontSize: '1.2em', "
-    "display: 'flex', alignItems: 'center'}}}}>"
-    "<img "
-    "src='https://github.githubassets.com/images/modules/logos_page/GitHub-Mark.png'"
-    " height='32px' width='32px' style={{{{marginRight: '10px'}}}}/>"
-    "View source on GitHub"
-    "</a>"
-    "</button>"
-    "</p>"
+    '{{< cta-button githubLink="{url}" >}}'
```

Comment:
haha, goodbye my old hacky button 😅

Reply:
It was most appreciated while it lasted  😺